qiskit/transpiler/passes/optimization/optimize_annotated.py

```diff
@@ -51,6 +63,7 @@ def __init__(
         equivalence_library: Optional[EquivalenceLibrary] = None,
         basis_gates: Optional[List[str]] = None,
         recurse: bool = True,
+        do_conjugate_reduction: bool = True,
```

Comment:
The main reason to have the longer name (I think) was to more clearly differentiate between the option of whether to apply conjugate reduction (currently called `do_conjugate_reduction`) and the actual method that does this (currently called `_conjugate_reduction`). The reason is that conjugate reduction is an option is because it's technically a more expensive optimization and we may want to have a way to switch it off in some cases.

Reply:
Yeah, it's not a big deal.  The internal private attribute could still be called `_do_conjugate_reduction` if you were worried about naming collisions while still permitting `OptimizeAnnotated(conjugate_reduction=False)`, but it's fine as-is.